src/containers/sidebars/data-global-sidebar/analyze-areas-sidebar-card/index.js

```diff
@@ -219,9 +219,8 @@ function AnalyzeAreasContainer(props) {
           : undefined;
 
       let layersToToggle = [];
-
-      if (newSelectedOption !== 'clear' && formerSelectedSlug !== undefined) {
-        layersToToggle.push({ layerId: formerSelectedSlug });
+      layersToToggle.push({ layerId: formerSelectedSlug });
+      if (newSelectedOption !== 'CLEAR' && formerSelectedSlug !== undefined) {
```

Comment:
Please bring back the former code or at least comment it if you are going to change it soon. Just not to leave unused code. Thanks!

Reply:
Sorry, thought i removed that new code, updated /removed it now